src/Components/Browser.JS/src/webpack.config.js

```diff
@@ -3,13 +3,13 @@ const webpack = require('webpack');
 
 module.exports = (env, args) => ({
     resolve: { extensions: ['.ts', '.js'] },
-    devtool: args.mode === 'development' ? 'inline-source-map' : 'none',
+    devtool: args.mode === 'development' ? 'source-map' : 'none',
```

Comment:
`inline-source-map` is problematic because I'd like to check in the Debug js too. Putting it in a separate source-map which is gitignored works reasonably well. If you happen to need it, you build Browser.JS locally and you get source maps. Does this seem reasonable @javiercn \ @SteveSandersonMS ?